app/models/report.rb

```diff
@@ -133,8 +74,8 @@ def self.expire(conditions = {})
     Log.joins(:report).where(:report_id => Report.where(cond)).delete_all
```

Comment:
This needs to use the type too, it's expiring logs which don't belong to this class.  I think removing `Report.` might do it.

<pre>
2015-10-12T13:40:54 [sql] [D]   SQL (3.1ms)  DELETE FROM "logs" WHERE "logs"."report_id" IN (SELECT "reports"."id" FROM "reports" WHERE (reports.created_at < '2015-10-09 11:40:54'))
2015-10-12T13:40:54 [sql] [D]   SQL (0.2ms)  DELETE FROM "messages" WHERE (id not IN (SELECT DISTINCT message_id FROM "logs" ))
2015-10-12T13:40:54 [sql] [D]   SQL (0.1ms)  DELETE FROM "sources" WHERE (id not IN (SELECT DISTINCT source_id FROM "logs" ))
2015-10-12T13:40:54 [sql] [D]   SQL (1.4ms)  DELETE FROM "reports" WHERE "reports"."type" IN ('TestReport') AND (reports.created_at < '2015-10-09 11:40:54')
</pre>